invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/MobileUploadsItem.js

```diff
@@ -38,7 +40,8 @@ export const MobileUploadsItem = ({
   ) : (
     <Icon name="upload" className="negative mr-10" />
   );
-
+  const uniqueViews = _get(result, "stats.all_versions.unique_views", -1);
```

Comment:
why -1 not 0 ? if -1 is displayed then it will be confusing for the users

Reply:
True! Didn't realize. Thanks!!